broker/lib/fabrik/CfPlatformManager.js

```diff
@@ -115,10 +115,14 @@ class CfPlatformManager extends BasePlatformManager {
   }
 
   buildSecurityGroupRules(options) {
+    let portRule = '1024-65535';
+    if (Array.isArray(options.applicationAccessPorts) && _.size(options.applicationAccessPorts) > 0) {
+      portRule = `${_.join(options.applicationAccessPorts,',')}`;
```

Comment:
Need not have the "``" here.... Just `_.join(options.applicationAccessPorts, ',')` suffices